scss/components/tweeting.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../node_modules/spinthatshit/src/loaders';
+@import '../colors';
+@import './images.scss';
+.tweeting-box {
+    display: flex;
+    justify-content: center;
+}
+.tweeting-roar {
+    padding-top: 5%;
```

Comment:
Since top/bottom left/right are often the same values you can write 

```scss
.tweeting-roar {
  padding: 5% 0;
}
```

So that you only have one value to edit if we want to change this.